common/utils/network.ts

```diff
@@ -26,3 +53,65 @@ export function getNetworkConfigFromId(
     return makeNetworkConfigFromCustomConfig(customConfig);
   }
 }
+
+type PathType = keyof DPathFormats;
+
+type DPathFormat =
+  | SecureWalletName.TREZOR
+  | SecureWalletName.LEDGER_NANO_S
+  | InsecureWalletName.MNEMONIC_PHRASE;
+
+export function getPaths(pathType: PathType): DPath[] {
+  const networkPaths: DPath[] = [];
+  Object.values(NETWORKS).forEach(networkConfig => {
```

Comment:
specifiying networkConfig as `NetworkConfig` in the function parameter will eliminate the need to do `path as DPath` on ln 69. This stops type coercion

Reply:
Typescript still didn't like this.